impactlab_tools/utils/paralog.py

```diff
@@ -56,9 +56,12 @@ def __init__(self, jobname, jobtitle, logdir, timeout, exclusive_jobnames=None):
                 self.logpath = logpath
                 break
 
-        # Record this process in the master log
-        with open(os.path.join(logdir, "master.log"), 'a') as fp:
-            fp.write("%s %s: %d %s\n" % (time.asctime(), self.jobtitle, os.getpid(), self.logpath))
+        try:
+            # Record this process in the master log
+            with open(os.path.join(logdir, "master.log"), 'a') as fp:
+                fp.write("%s %s: %d %s\n" % (time.asctime(), self.jobtitle, os.getpid(), self.logpath))
+        except:
+            print "Warning: Could not append to master log."
```

Comment:
if you change this to `print(...)` this will solve your current errors

Reply:
any reason you're printing rather than using warnings.warn?

Reply:
Not much of a reason.  This is almost always going into log files (the prints I mean, aside from that the paralog results themselves go into logs), and I like lines that say `Warning: ...` rather than `<code line>: UserWarning: ...`.  But I don't really care.

Reply:
cool. just curious.

Reply:
looks like there was another `print ` line in versions - I made the fix. let's see if this passes 👍